lib/scala/json-rpc-server-test/src/main/scala/org/enso/jsonrpc/test/JsonRpcServerTestKit.scala

```diff
@@ -174,10 +174,12 @@ abstract class JsonRpcServerTestKit
     def fuzzyExpectJson(
       json: Json,
       timeout: FiniteDuration = 5.seconds.dilated
-    )(implicit pos: Position): Assertion = {
+    )(implicit pos: Position): Json = {
```

Comment:
Shouldn't we leave `Either` here?

Reply:
If the JSON does not match the expected `json` value, the assertion will fail and the function will fail with the assertion exception